src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Organisations]
 (
-	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
-    [Name] VARCHAR(255) NOT NULL, 
-    [OdsCode] VARCHAR(8) NULL, 
-    [LastUpdated] DATETIME2 NOT NULL
-)
+	Id UNIQUEIDENTIFIER NOT NULL,
```

Comment:
The name column has brackets but the others don't. Could we make this consistent?

Reply:
The reason for the brackets around `name` is just because VS will highlight it as a reserved word (even though it isn't). We can remove the brackets here but I suggest we just leave them in this case as there could be other cases if we ever use a reserved word for a column name but hopefully we will avoid that... However, in all other cases DDL looks better without square brackets in my opinion.